packages/mongodb-chatbot-ui/src/ModalView.tsx

```diff
@@ -220,11 +229,7 @@ export function ModalView(props: ModalViewProps) {
                     onChange: (e) => {
                       setInputText(e.target.value);
                     },
-                    placeholder: conversation.error
-                      ? "Something went wrong. Try reloading the page and starting a new conversation."
-                      : awaitingReply
-                      ? "MongoDB AI is answering..."
-                      : "Ask MongoDB AI a Question",
+                    placeholder: inputPlaceholder,
```

Comment:
Nope it's baked into the MongoDbInputBarPlaceholder https://github.com/mongodb/chatbot/pull/289/files

Though I suppose we could override the user's custom placeholder if there's an error. The issue there is that they can't define a custom one then, eg if you should do something other than refresh the page.

Reply:
re custom err msgs, i guess it could be nice to have it be configurable, but seems relatively low priority. maybe we could just leave a TODO comment for it if you don't want to add the functionality now.

Reply:
Added `fatalErrorMessage` props to override the default